returnn/torch/data/pipeline.py

```diff
@@ -268,9 +271,13 @@ def _parse_max_seqs(
     def _get_user_func_kwargs_from_data_dict(data_dict: Dict[str, Any]) -> Dict[str, Any]:
         epoch = int(data_dict["epoch"])
         seq_idx = int(data_dict["seq_idx"])
-        num_seqs = int(data_dict["num_seqs"])  # >=1 if known, otherwise -1
-        epoch_continuous = (epoch - 1 + (seq_idx + 1) / num_seqs) if num_seqs > 0 else None
-        return {"epoch": epoch, "seq_idx": seq_idx, "epoch_continuous": epoch_continuous, **get_fwd_compat_kwargs()}
+        complete_frac = float(data_dict["complete_frac"])  # >= 0 if known, otherwise -1
+        return {
+            "epoch": epoch,
+            "complete_frac": complete_frac if complete_frac >= 0.0 else None,
```

Comment:
You are breaking here user code? That doesn't make sense?

Reply:
Same as above.